rclcpp/src/rclcpp/executor.cpp

```diff
@@ -730,6 +736,7 @@ Executor::wait_for_work(std::chrono::nanoseconds timeout)
 
   rcl_ret_t status =
     rcl_wait(&wait_set_, std::chrono::duration_cast<std::chrono::nanoseconds>(timeout).count());
+
```

Comment:
nit: revert this unrelated whitespace change.

Reply:
```suggestion
```

Reply:
Fixed in https://github.com/ros2/rclcpp/pull/1505/commits/44524078585283875a7a7cf434f3f601e5f98af6.